src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -943,6 +942,31 @@ void LottieParser::parseAssets()
     }
 }
 
+LottieMarker* LottieParser::parseMarker()
+{
+    enterObject();
+    
+    auto marker = new LottieMarker;
+    
+    while (auto key = nextObjectKey()) {
+        if (!strcmp(key, "cm")) marker->name = getStringCopy();
+        else if (!strcmp(key, "tm")) marker->time = getInt();
+        else if (!strcmp(key, "dr")) marker->duration = getInt();
+        else skip(key);
+    }
+    
+    return marker;
+}
+
+void LottieParser::parseMarkers()
+{
+    enterArray();
+    while (nextArrayValue())
+    {
```

Comment:
You can check original code style for while-loop.
`while (nextArrayValue()) {` (no newline in brace)